### PR TITLE
[BTS-332] Snackbar

### PR DESCRIPTION
**Pull request by ggdaltoso**

### CODE REVIEW

**Comment by rapahaeru** [comment]
Something is wrong with the button proptypes. 
Captura de tela de 2019-05-27 10-22-54

File: components/SnackBar/SnackBar.jsx [hunk 1]
```diff
@@ -1,39 +1,91 @@
...
+      background: backgroundInverted,
+      text: textInverted,
+    };
+  }
+
+  return {
+    background,
+    text,
+  };
+};
```
Comment (ddsilva) [inline components/SnackBar/SnackBar.jsx]:
```suggestion
const getBackgroundAndTextColorBySkin = ({
  skin,
  theme: {
    components: {
      snackbar: {
        skins: {
          [skin]: { text, background },
        },
      },
    },
  },
  inverted,
}) => ({
  background: inverted ? text : background,
  text: inverted ? background : text
});
```

**Pull request merged**